src/components/DetailsArtifacts/DetailsArtifactsView.js

```diff
@@ -55,9 +63,12 @@ const DetailsArtifactsView = ({
               <Tooltip template={<TextTooltipTemplate text="Show Details" />}>
                 <Link
                   target="_blank"
-                  to={`/projects/${
-                    match.params.projectName
-                  }/artifacts/${artifact.db_key || artifact.key}/overview`}
+                  to={
+                    artifactScreenLinks[artifact.kind] ||
```

Comment:
If the expression `artifactScreenLinks[artifact.kind]` resulted in a string value I‘d say that it could justify using `||` operator here, since it works for the empty string `''`, too.
But since this expression is either an object or `undefined` — I think `??` is better suited here.

```suggestion
                    artifactScreenLinks[artifact.kind] ??
```